### PR TITLE
Update programs.php

### PR DESCRIPTION
**Pull request by cdavis-ubc**
- changed the modal links to class=fancybox-inline because data-toggle=modal is stripped in browser
- added sanitize_text_field to departments cf to strip <a></a> from grid box

### CODE REVIEW

File: templates/education/programs.php [hunk 1]
```diff
@@ -66,20 +71,21 @@
...
 				<div class="iso-title">
-					<?php the_title(); ?>
+					<?php echo esc_html( $the_title ); ?>
 				</div>
 
 				<div class="iso-description hidden">
 					<?php the_content(); ?>
-					<?php echo esc_html( $department ); ?>
-					<?php echo esc_html( $location ); ?>
+					<?php echo esc_attr( $department ); ?>
```
Comment (richardtape) [inline templates/education/programs.php]: Why `esc_attr()` for these? They're not attributes?

File: templates/education/programs.php [hunk 2]
```diff
@@ -66,20 +71,21 @@
...
+					<?php echo esc_attr( $location ); ?>
+					<?php echo esc_html( $the_title ); ?>
 				</div>
 
 				<div class="excerpt">
 
 					<div class="location">
 						<?php echo esc_html( $location ); ?>
-						<small><?php echo esc_html( $department ); ?></small>
+						<small><?php echo sanitize_text_field( $department ); ?></small>
```
Comment (richardtape) [inline templates/education/programs.php]: Sanitization is for input, not output. `wp_kses_post()` may be more appropriate?

**Comment by richardtape** [comment]
Can't merge this one in as it stands as the branch has conflicts. Can you let me know when you've pulled from master and resolved the conflicts, then push back to this branch.

**Comment by richardtape** [comment]
Any movement on this, @cdavis-ubc

**Comment by cdavis-ubc** [comment]
Yes I put in a request to have my master updated from ubc master.  Did you get the request? I need my master updated with latest commits before I can send pull request with my update. It is the pull request #2 "Get latest commits for ubc-use-template-shortcode" 
Thanks.

**Comment by richardtape** [comment]
Your master is YOUR master. A pull request goes from YOU to ME not the 
other way around. You'll need to update your fork with any changes from 
the main UBC branch.

> cdavis-ubc mailto:notifications@github.com
> September 30, 2015 at 12:28 PMvia Postbox 
> https://www.postbox-inc.com/?utm_source=email&utm_medium=sumlink&utm_campaign=reach
> 
> Yes I put in a request to have my master updated from ubc master. Did 
> you get the request? I need my master updated with latest commits 
> before I can send pull request with my update. It is the pull request 
> #2 https://github.com/ubc/ubc-use-template-shortcode/pull/2 "Get 
> latest commits for ubc-use-template-shortcode"
> Thanks.
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/ubc/ubc-use-template-shortcode/pull/3#issuecomment-144514823.
> 
> cdavis-ubc mailto:notifications@github.com
> September 23, 2015 at 1:58 PMvia Postbox 
> https://www.postbox-inc.com/?utm_source=email&utm_medium=sumlink&utm_campaign=reach
> - changed the modal links to class=fancybox-inline because
>   data-toggle=modal is stripped in browser
> - added sanitize_text_field to departments cf to strip from grid box
> 
> ---
> 
> ```
>     You can view, comment on, or merge this pull request online at:
> ```
> 
> https://github.com/ubc/ubc-use-template-shortcode/pull/3
> 
> ```
>     Commit Summary
> ```
> - Update programs.php
>   
>   ```
>   File Changes
>   ```
> - _M_ templates/education/programs.php
>   https://github.com/ubc/ubc-use-template-shortcode/pull/3/files#diff-0
>   (45)
>   
>   ```
>   Patch Links:
>   ```
> - https://github.com/ubc/ubc-use-template-shortcode/pull/3.patch
> - https://github.com/ubc/ubc-use-template-shortcode/pull/3.diff
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/ubc/ubc-use-template-shortcode/pull/3.

## 

Sent from Postbox 
https://www.postbox-inc.com/?utm_source=email&utm_medium=siglink&utm_campaign=reach

**Comment by cdavis-ubc** [comment]
I realize that. That's what I was trying to do. Update MY master from the UBC master. So I sent a request to do that. It reads: 
"cdavis-ubc wants to merge 28 commits into cdavis-ubc:master from ubc:master"  
so I thought this was going from you to me. 
If that is not the way to get the 28 commits to my forked master then can I do it manually?

**Comment by richardtape** [comment]
I have no control over yours. Your fork is yours. You can do whatever 
you want with your fork :)

If you look at the message it's going FROM ubc:master (i.e. "me") INTO 
cdavis-ubc:master (i.e. "you"). (Although it could be clearer, it's 
worded weirdly).

R

cdavis-ubc wrote:

**Comment by cdavis-ubc** [comment]
OK thanks for clarifying  :)
I'll work on that now and get back to you when ready.

**Comment by richardtape** [comment]
no probs. I'm just going through the PRs now. I've commented on a few, 
others I've merged in already and will pull them onto verf shortly.

cdavis-ubc wrote:

**Pull request closed without merging**